api/guild.go

```diff
@@ -259,6 +259,11 @@ func (g *Guild) IconURL(size int) *string {
 	return &u
 }
 
+// GetAuditLogs gets AuditLog(s) for this Guild
+func (g *Guild) GetAuditLogs(userID Snowflake, actionType AuditLogEvent, before Snowflake, limit int) (*AuditLog, restclient.RestError) {
```

Comment:
Shouldn't this return more than one?

Reply:
nope
that is just a wrapper struct as described in the docs <https://discord.com/developers/docs/resources/audit-log#get-guild-audit-log>